flw/src/main/java/org/motechproject/nms/flw/service/impl/FrontLineWorkerImportServiceImpl.java

```diff
@@ -131,13 +132,26 @@ private FrontLineWorker flwFromRecord(Map<String, Object> record, State state) {
             // if the mcts id has changed.
             if (flw != null && mctsFlwId != null && !mctsFlwId.equals(flw.getMctsFlwId())) {
                 throw new CsvImportDataException(String.format("Existing FLW with same MSISDN (%s) but " +
-                                        "different MCTS ID (%s != %s)", msisdn, mctsFlwId, flw.getMctsFlwId()));
+                                        "different MCTS ID (%s != %s)", obscureNumber(msisdn), mctsFlwId, flw.getMctsFlwId()));
             }
         }
 
         return flw;
     }
 
+    // obscure code from LogHelper in API package. Copy-paste since we cannot do a circular reference
```

Comment:
Okey dokey, LGTM!

On Mon, Oct 5, 2015 at 4:44 PM, kosh <notifications@github.com> wrote:

> In
> flw/src/main/java/org/motechproject/nms/flw/service/impl/FrontLineWorkerImportServiceImpl.java
> <https://github.com/motech-implementations/mim/pull/609#discussion_r41211913>
> :
>
> >              }
> >          }
> >
> >          return flw;
> >      }
> >
> > +    // obscure code from LogHelper in API package. Copy-paste since we cannot do a circular reference
>
> This felt like the best of all stop gaps
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/609/files#r41211913>.
>



-- 
Frank Huster
Mobile Health Innovations
Grameen Foundation
+1.206.325.6690 x201